### PR TITLE
README for OPA Gatekeeper

### PR DESCRIPTION
**Pull request by igorkchyts**

### CODE REVIEW

**Comment by igorkchyts** [comment]
Closes #17

File: open-policy-agent/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# OPA Gatekeeper policies
```
Comment (michaelkotelnikov) [inline open-policy-agent/README.md:1]: Add a capital **P** in `policies`

File: open-policy-agent/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+------- | ----------- | -------------
+No policies yet       |  |
+
+### Trusted Image Sources
+Policy  | Description | Prerequisites
+------- | ----------- | -------------
+[disallowedtags](./trusted-image-sources/disallowedtags) | Ensures that images do not contain a pre-defined tag (by default, the policy disables the `latest` tag) | 
+
+## Applying Policies
+The policies can be created by applying the `template.yaml` and `contraint.yaml` provided in each policy directory under specific security control.
```
Comment (michaelkotelnikov) [inline open-policy-agent/README.md:66]: Would change to -

```
The policies can be created by applying the the custom resources defined in the `template.yaml` and `contraint.yaml` files to an OpenShift cluster. The files are provided in each policy directory under specific security control.
```

**Comment by michaelkotelnikov** [comment]
@igorkchyts  Added some comments.

**Review by michaelkotelnikov — APPROVED** [review]

**Pull request merged**